### PR TITLE
Update Dependencies

### PR DESCRIPTION
**Pull request by cassidycodes**
I'm working on a PR to restrict the usage of RecursiveOpenStruct and I noticed that a few dependencies need to get updated.

-----------------

Before submitting the PR make sure the following are checked:

* [x] Wrote [good commit messages][1].
* [ ] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [x] Added tests.
* [x] Added an entry to the [Changelog](https://github.com/wealthsimple/rubocop-vendor/blob/master/CHANGELOG.md) if the new code introduces user-observable changes. See [changelog entry format](https://github.com/wealthsimple/rubocop-vendor/blob/master/CONTRIBUTING.md#changelog-entry-format).
* [x] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences.
* [x] Run `bundle exec rake default`. It executes all tests and RuboCop for itself, and generates the documentation.

[1]: https://chris.beams.io/posts/git-commit/

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.6.3 - 2021-1-29
```
Comment (pfeffer) [inline CHANGELOG.md:8]:
```suggestion
## 0.6.2 - 2021-1-29
```

**Review by pfeffer — APPROVED** [review]

File: lib/rubocop/cop/vendor/rollbar_inside_rescue.rb [hunk 2]
```diff
@@ -29,7 +29,7 @@ module Vendor
       #     end
       #   end
       #
-      class RollbarInsideRescue < Cop
+      class RollbarInsideRescue < Base
```
Comment (pfeffer) [inline lib/rubocop/cop/vendor/rollbar_inside_rescue.rb:32]: curious, why was this change needed?

Comment (cassidycodes) [reply lib/rubocop/cop/vendor/rollbar_inside_rescue.rb:32]: RuboCop has a rule that enforces this. I don't fully know why because the comment in the code links to a page that 404s. But I think that RuboCop::Cop::Base is meant to be easier to work with.

https://github.com/rubocop/rubocop/blob/4fcf0ffb942b1d37e646e4230e7f1a8802feb9ca/lib/rubocop/cop/cop.rb#L8-L10

Comment (cassidycodes) [reply lib/rubocop/cop/vendor/rollbar_inside_rescue.rb:32]: Ah, my comment got lost because I rebased and force pushed. Posting here as well in case anyone is curious:

RuboCop has a rule that enforces this. I don't fully know why because the comment in the code links to a page that 404s. But I think that RuboCop::Cop::Base is meant to be easier to work with.

https://github.com/rubocop/rubocop/blob/4fcf0ffb942b1d37e646e4230e7f1a8802feb9ca/lib/rubocop/cop/cop.rb#L8-L10

**Review by pfeffer — APPROVED** [review]
🔥

**Review by cabello — APPROVED** [review]
Looks very fresh! :shipit:  Let me know if you are able to release this gem, hopefully this is not tied to my personal account.
